mvp-spec/operator-client.md

```diff
@@ -0,0 +1,576 @@
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+```json
+{
+  "body": {
+    "identifiers": [
+      {
+        "version": "0.1",
+        "type": "paf_browser_id",
+        "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
+        "source": {
+          "domain": "operator.paf-operation-domain.io",
+          "timestamp": 1642504380,
+          "signature": "ZtvL2PAZRUs0A3W+Af1Vj/8qNbZV5pHGJWTqc+frfEBwiLed08xY5dpgsPHelLG3f5tV39tKPEKhcXD+hJA3eQ=="
+        }
+      }
+    ],
+    "preferences": {
+      "version": "0.1",
+      "data": {
+        "use_browsing_for_personalization": true
+      },
+      "source": {
+        "domain": "cmp.com",
+        "timestamp": 1642504560,
+        "signature": "uOFJJo3xpyp5goiF9gHh10NqhwIlnEEVeyke2OtPseXPq+BSC/R/3LgqmE8b3TYqFbn2HvKDx0/arFwRBCURLg=="
+      }
+    }
+  },
+  "sender": "operator.paf-operation-domain.io",
+  "receiver": "advertiser.com",
+  "timestamp": 1643041150,
+  "signature": "IchR+WB5ekkmm+ERNju0fc3pVHpcOLA5IKn2HGsYJusH98yCZWlLg9dNzNdC2nAPc0OGcHgHqRcZc38S9ckO0w=="
+}
+```
+<!--partial-end-->
+
+- and is used as the **POST payload** to the following call:
+
+<!--partial-begin { "files": [ "verifyGetIdsPrefs.http" ], "block": "http" } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+```http
+POST /paf-proxy/v1/verify/read
+Host: cmp.com
+```
+<!--partial-end-->
+
+- response in case the signature is valid: same as request
+
+<!--partial-begin { "files": [ "getIdsPrefsResponse_known.json" ], "block": "json" } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+```json
+{
+  "body": {
+    "identifiers": [
+      {
+        "version": "0.1",
+        "type": "paf_browser_id",
+        "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
+        "source": {
+          "domain": "operator.paf-operation-domain.io",
+          "timestamp": 1642504380,
+          "signature": "ZtvL2PAZRUs0A3W+Af1Vj/8qNbZV5pHGJWTqc+frfEBwiLed08xY5dpgsPHelLG3f5tV39tKPEKhcXD+hJA3eQ=="
+        }
+      }
+    ],
+    "preferences": {
+      "version": "0.1",
+      "data": {
+        "use_browsing_for_personalization": true
+      },
+      "source": {
+        "domain": "cmp.com",
+        "timestamp": 1642504560,
+        "signature": "uOFJJo3xpyp5goiF9gHh10NqhwIlnEEVeyke2OtPseXPq+BSC/R/3LgqmE8b3TYqFbn2HvKDx0/arFwRBCURLg=="
+      }
+    }
+  },
+  "sender": "operator.paf-operation-domain.io",
+  "receiver": "advertiser.com",
+  "timestamp": 1643041150,
+  "signature": "IchR+WB5ekkmm+ERNju0fc3pVHpcOLA5IKn2HGsYJusH98yCZWlLg9dNzNdC2nAPc0OGcHgHqRcZc38S9ckO0w=="
+}
+```
+<!--partial-end-->
+
+- response in case the signature is **not** valid:
+
+<!--partial-begin { "files": [ "verifyGetIdsPrefs_invalid.json" ], "block": "json" } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+```json
+{
+  "message": "Invalid signature"
+}
+```
+<!--partial-end-->
+
+</details>
+
+## Frontend operator client
+
```

Comment:
Here, I plan to add a diagram that explains when the frontend operator client calls the backend operator proxy.
(for another PR)